### ISSUE TITLE
Fixable tooling

### ISSUE THREAD

**Issue by Youssef1313** [DavidAnson/markdownlint#319]
Many rules in Rules.md explanation states the following:

> Fixable: Most violations can be fixed by tooling

it would be good to provide link of this tooling if there is a popular one.

**Comment by DavidAnson** [comment]
Links are at the top of the README: https://github.com/DavidAnson/markdownlint#related

**Comment by Youssef1313** [comment]
@DavidAnson Thanks for the incredibly fast answer. The tooling under the "related" heading seems to have checkers, not fixers (unless I'm missing something or not fully understanding the given toolings). Can you please provide more info?

Thanks in advance.

**Comment by Youssef1313** [comment]
I found [this](https://github.com/DavidAnson/vscode-markdownlint/blob/main/README.md#rules) very helpful. I didn't know that VS code extension is able to fix these. I think it would be a better experience that you add this link to Rules.md file in this repo.

**Comment by DavidAnson** [comment]
`markdownlint-cli` and `markdownlint-cli2` will both fix multiple files at once.

If you can demonstrate a scenario where the VS Code extension is fixing a rule that is disabled, please let me know because I would like to fix that. Disabled rules should not be reported in the first place, and therefore not be fixable. If you are seeing a rule reported, then it has not been disabled correctly.￼

**Comment by Youssef1313** [comment]
A file with only the following content:

```markdown
Test
-----

will be transformed into:

```markdown
## Test
```

on saving.

My .markdownlint.json is:

```
{
"default": false
}
```

**Comment by DavidAnson** [comment]
That is not a change `markdownlint` will make. Please check to see if you are using the Prettier extension or something like that.

**Comment by Youssef1313** [comment]
Great. Really thanks.
